### ISSUE TITLE
Prompt for the generator name

### ISSUE THREAD

**Issue by arnaud** [jondot/hygen#161]
#### Is it possible to prompt the user for the generator name?

I actually can't figure out to make it work:

`_templates/app/new/index.js:`
```
module.exports = {
  prompt: async ({ prompter, args }) => {
    let prompt
    prompt = await prompter.prompt({
      type: 'input',
      name: 'name',
      initial: args.name,
      message: `Your app name`
    })
    return prompt
 })
}
```

When executing it from the console, I get:

```
$ hygen app new
? Your app name > john
x   exists: packages/unnamed/.babelrc. Overwrite? (y/N)
```

Expected name: john
Actual name: unnamed

Any help would be very much appreciated :)

**Comment by arnaud** [comment]
my bad... its was as a matter of fact a very simple matter: the above code snippet works. What didn't was overriding the `prompt` variable again and again, thus losing track of its previously-set key/value pairs.

TL;DR: nevermind, my bad
